packages/build/tests/blobs_upload/fixtures/src_with_malformed_blobs_metadata/build.mjs

```diff
@@ -0,0 +1,8 @@
+import { mkdir, writeFile } from 'node:fs/promises'
+
+await mkdir('dist/.netlify/blobs/deploy', { recursive: true })
+
+await Promise.all([
+  writeFile('dist/.netlify/blobs/deploy/with-metadata.txt', 'another value'),
+  writeFile('dist/.netlify/blobs/deploy/$with-metadata.txt.json', 'this is not json'),
+])
```

Comment:
I've changed it to write files this way instead of using cp since cp is not available in Node 14 and was failing CI tests